python/ray/train/accelerators/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .base import Accelerator  # noqa: F401
+from .torch import TorchAccelerator  # noqa: F401
```

Comment:
Word of caution since I ran into this issue recently, I think having this here would raise an error if the user doesn't have `torch` installed.

Reply:
Fixed